application/src/main/kotlin/org/gxf/soapbridge/monitoring/MonitoringService.kt

```diff
@@ -0,0 +1,29 @@
+package org.gxf.soapbridge.monitoring
+
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Timer
+import org.springframework.stereotype.Service
+import java.time.Duration
+import java.time.Instant
+
+@Service
+class MonitoringService(
+    private val registry: MeterRegistry
+) {
+
+    companion object {
+        private const val METRIC_PREFIX = "gxf.soap.bridge"
+    }
+
+    fun recordConnectionTime(startTime: Instant, context: String, successful: Boolean) {
```

Comment:
Ik zou hier een naam kiezen die meer zegt in de richting van 'roundtrip' of 'response' over de proxy. Ja, de SOAP connection hebben we ook al die tijd open, maar dat is volgens mij niet wat we hier meten